api/api.proto

```diff
@@ -1807,21 +1816,21 @@ service OpenStoragePolicy {
       };
     }
 
-  // Enforces specified storage policy
-  rpc Enforce(SdkOpenStoragePolicyEnforceRequest)
-    returns (SdkOpenStoragePolicyEnforceResponse) {
+  // Set specified storage policy as default policy
+  rpc SetDefault(SdkOpenStoragePolicySetDefaultRequest)
```

Comment:
Changing and deleting an API is quite disruptive in the SDK. We need to make sure this doesn't happen again because non-openstorage developers of 2.0.x will see missing APIs when moving to 2.1.

This is ok for now, since it has been here a short time.

Reply:
just curious does api.proto support to declare some API's are deprecated or identityManager is only way

Reply:
Good question. In April, we will be coming up with a process of deprecating APIs, and also creating new Alpha-Beta-GA releases for APIs.